packages/theme/package.json

```diff
@@ -36,7 +36,9 @@
   "dependencies": {
     "@oclif/core": "2.11.7",
     "@shopify/cli-kit": "3.49.1",
-    "@shopify/theme-language-server-node": "1.1.0"
+    "@shopify/theme-check-node": "1.15.0",
```

Comment:
IIUC, we're planning to update the version here to include https://github.com/Shopify/theme-tools/pull/158. Is that correct? :)